lib/qubicInterface.ts

```diff
@@ -93,6 +149,38 @@ export class QubicInterface {
     };
   }
 
+  async getTransactionWithPayload(
+    sourceSeed: string,
+    destinationPublicId: string,
+    amount: number,
+    tick: number,
+    inputType: number,
+    payload: Uint8Array,
+    payloadPackageSize: number,
+    asBase64: boolean = true
```

Comment:
@Qubic-Hub  do we need payloadPackageSize as param? can't get it as length of the payload?